src/pages/Projects/components/RoleModal/RoleModal.tsx

```diff
@@ -355,6 +355,7 @@ export default function RoleModal({
 
       mutate(`/projects/${project?.id}`);
     } catch (e) {
+      console.error(e);
```

Comment:
doing something in the catch block other than throwing the error makes the linter happy